### PR TITLE
Fix up indentation where it got messed up in previous merges.

### PR DESCRIPTION
**Pull request by gafter**
Stub IOperation for pattern-matching operations (to avoid crashes).

Please review. @jaredpar @AlekseyTs

### CODE REVIEW

File: src/Compilers/CSharp/Portable/BoundTree/Expression.cs [hunk 1]
```diff
@@ -2760,13 +2760,12 @@ partial class BoundIsPatternExpression
         public override void Accept(OperationVisitor visitor)
         {
             // TODO: implement IOperation for pattern-matching constructs (https://github.com/dotnet/roslyn/issues/8699)
-            throw new NotImplementedException();
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Portable/BoundTree/Expression.cs]: Shouldn't this method do something useful?

**Comment by AlekseyTs** [comment]
If bound node returns particular OperationKind, it should implement an interface matching this kind. None of the modified pattern matching nodes do this. I think, for now, they all should return Invalid kind and implement corresponding interface. Accept methods should be implemented accordingly. 

There should be tests.

**Comment by AlekseyTs** [comment]
Another option is to return OperationKind.None for all new nodes for now, and visit them accordingly in Accept.

File: src/Compilers/CSharp/Portable/BoundTree/Expression.cs (hunk 1 above)
Comment (gafter) [inline src/Compilers/CSharp/Portable/BoundTree/Expression.cs]: Yes, once the "something useful" is defined for IOperation. That is why there is a reference to issue https://github.com/dotnet/roslyn/issues/8699 here.

**Comment by gafter** [comment]
@AlekseyTs Previous review comments addressed now.

**Comment by AlekseyTs** [comment]
LGTM

**Pull request merged**